README.md

```diff
@@ -2,10 +2,10 @@
 A wrapper in Node for the compiled protoc from https://github.com/protocolbuffers/protobuf.
 
 ## Version
-It's currently using Protocol Buffers `v3.20.3`.
+It's currently using Protocol Buffers `v21.9`.
```

Comment:
This won't work. See https://github.com/YePpHa/node-protoc/pull/7#issue-1428352169, why.